phpBB/adm/style/detailed_message_body.html

```diff
@@ -0,0 +1,17 @@
+<!-- INCLUDE overall_header.html -->
+
+<div {% if S_USER_NOTICE %}class="successbox"{% else %}class="errorbox"{% endif %}>
+	<h3>{{ MESSAGE_TITLE }}</h3>
+	<p>{{ MESSAGE_TEXT }}</p>
+</div>
+
+<div style="overflow-x:scroll">
+	<fieldset>
```

Comment:
To get a console look, add a class to this element with:
```css
background-color: #2a2a2a;
color: #f1f1f1;
font-family: monospace;
```
Also remove the legend below